File: 1-two-sum/1-two-sum.java
class Solution {
public int[] twoSum(int[] nums, int target) {
        Map<Integer,Integer> hs=new HashMap<Integer,Integer>();
        for(int i=0;i<nums.length;i++){
            int sum=target-nums[i];
            if(hs.containsKey(sum)){
                return new int[]{hs.get(sum),i};
            }
            hs.put(nums[i],i);
        }
        return new int[]{0,0};
    }
}

File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        si={}
        for i,n in enumerate(nums):
            if((target-n) in si):
                return [si[target-n],i]
            si[n]=i;
        return [0,0]
        